packages/arb-ts/src/lib/bridge_helpers.ts

```diff
@@ -210,20 +215,37 @@ export class BridgeHelper {
 
   static getUpdateTokenInfoEventResult = async (
     l1Transaction: providers.TransactionReceipt,
```

Comment:
these event functions are a bit silly and should obviously by dryed up; with ts-generics magic we can preserve the dynamic returned interfaces I think. But I'd rather do that in a subsequent PR